RecyclerView/app/src/main/java/net/boondockradio/recyclerview/StatesAdapter.java

```diff
@@ -28,24 +34,44 @@ public ItemViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
 
     @Override
     public void onBindViewHolder(ItemViewHolder holder, final int position) {
-        String stateText;
-        stateText = states.get(position);
+        final StateItem item = states.get(position);
+        String stateText = item.title;
 
         final Context context = holder.itemView.getContext();
         holder.textView.setText(stateText);
 
+        TypedArray images = context
+                .getResources()
+                .obtainTypedArray(R.array.states_images);
```

Comment:
then adding a state will look like this:

`states.add(new StateItem("Alabama", R.drawable.alabama));`